phylonco-lphy/src/test/java/phylonco/lphy/evolution/copynumbermodel/ReadCopyProfileTest.java

```diff
@@ -0,0 +1,178 @@
+package phylonco.lphy.evolution.copynumbermodel;
+
+import lphy.base.evolution.Taxa;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class ReadCopyProfileTest {
+
+    private ReadCopyProfile reader;
+
+    @BeforeEach
+    void setUp() {
+        reader = new ReadCopyProfile(new lphy.core.model.Value<>("dummy", "dummy.txt"));
+    }
+
+    @Test
+    void testParseGinkgoFormat_WithRealData() {
+        String realFilePath = "/Users/ming/Desktop/Aug22/CRC09-GINKGO_24cells.CN.txt";
```

Comment:
We should use a small example with dummy data (not real data), and replace this with a relative path starting from the repository root.